### PR TITLE
Adds Devd Formula

### PR DESCRIPTION
**Pull request by gcoonrod**
Adds Formula for [devd](https://github.com/cortesi/devd), a lightweight http server for developers.

### CODE REVIEW

File: Library/Formula/devd.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        system 'go', 'get', 'github.com/GeertJohan/go.rice'
+        system 'go', 'get', 'github.com/toqueteos/webbrowser'
+        system 'go', 'get', 'github.com/alecthomas/kingpin'
+
+        system 'go', 'build', '-o', 'devd'
+        bin.install 'devd'
+    end
+
+    test do
+        system "#{bin}/devd", '--version'
```
Comment (mikemcquaid) [inline Library/Formula/devd.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/devd.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+require 'language/go'
+
+class Devd < Formula
+    desc 'A http daemon for local development'
```
Comment (mikemcquaid) [inline Library/Formula/devd.rb]: Use two-character tabs rather than spaces.

Comment (mikemcquaid) [inline Library/Formula/devd.rb]: Also use double-quotes everywhere (generally run `brew audit --strict --online`)

File: Library/Formula/devd.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+require 'language/go'
```
Comment (bfontaine) [inline Library/Formula/devd.rb]: Please use double quotes everywhere.

**Comment by mikemcquaid** [comment]
Passing on this for now as it's less than a day old, sorry!

File: Library/Formula/devd.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    depends_on 'go' => :build
+
+    def install
+        ENV['GOPATH'] = buildpath
+
+        # Install Go dependencies
+        system 'go', 'get', 'github.com/cortesi/devd'
+        system 'go', 'get', 'github.com/GeertJohan/go.rice'
+        system 'go', 'get', 'github.com/toqueteos/webbrowser'
+        system 'go', 'get', 'github.com/alecthomas/kingpin'
```
Comment (bfontaine) [inline Library/Formula/devd.rb]: Please use resources instead of `go get`ing them.

**Pull request closed without merging**